databallpy/load_data/tracking_data/tracab.py

```diff
@@ -0,0 +1,199 @@
+from typing import Tuple
+
+import bs4
+import numpy as np
+import pandas as pd
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+from databallpy.load_data.metadata import Metadata
+from databallpy.load_data.tracking_data._add_player_data_to_dict import _add_player_data_to_dict
+from databallpy.load_data.tracking_data._add_ball_data_to_dict import _add_ball_data_to_dict
+from databallpy.load_data.tracking_data._insert_missing_rows import _insert_missing_rows
+from databallpy.load_data.tracking_data._to_match_time import _to_match_time
+
+def load_tracab_tracking_data(
+    tracab_loc: str, meta_data_loc: str, verbose: bool = True
+) -> Tuple[pd.DataFrame, Metadata]:
+    """Function to load tracking data and metadata from the tracab format
+
+    Args:
+        tracab_loc (str): location of the tracking_data.dat file
+        meta_data_loc (str): location of the meta_data.xml file
+        verbose (bool): whether to print on progress of loading in the terminal , defaults to True
+
+    Returns:
+        Tuple[pd.DataFrame, Metadata], the tracking data and metadata class
+    """
+
+    tracking_data = _get_tracking_data(tracab_loc, verbose)
+    metadata = _get_metadata(meta_data_loc)
+
+    tracking_data["match_time_td"] = [_to_match_time(x, metadata) for x in tqdm(tracking_data["timestamp"])]
```

Comment:
This loop is unnecessary. Maybe just input the timestamp series and the metadata and get a match time series back: `tracking_data["match_time_td"] = _to_match_time(tracking_data["timestamp"], metadata)`